cmd/account/console.go

```diff
@@ -104,6 +117,16 @@ func (o *consoleOptions) run() error {
 
 	ctx := context.TODO()
 	var accountID string
+	if o.clusterID != "" {
+		accountClaim, err := k8s.GetAccountClaimFromClusterID(ctx, o.kubeCli, o.clusterID)
+		if err != nil {
+			return err
+		}
+		if accountClaim == nil {
+			return fmt.Errorf("Could not find any accountClaims for cluster with ID: %s", o.clusterID)
+		}
+		o.accountName = accountClaim.Spec.AccountLink
```

Comment:
Not a major thing, but if the accounClaim has not been claimed yet, the AccountLink could be empty. Should we return an error here?

If we continue it will trigger a weird error below, because we'll end up getting the AWS Account ID (which is empty as well)